source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Ember 3.13 Released (Octane Preview)
+author: Kenneth Larsen, Tom Dale, Yehuda Katz
+tags: Releases, 2019, 3, 3.13, Version 3.x
+responsive: true
+---
+
+Today, the Ember project is releasing version 3.13 of Ember.js, Ember Data, and Ember CLI.
+
+Ember 3.13 is the final release of Ember before Octane. Ember 3.13 is also the Octane preview. The preview allows adventurous community members to help us put the final round of polish on the edition before Octane becomes official in 3.14.
+
+Additionally, version 3.12 of Ember is now promoted to LTS (Long Term Support). An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS version of Ember was 3.8.
+
+## The Octane Preview
+
+The Octane features in 3.13 are stable public APIs with our normal SemVer guarantees. If you build an app using APIs in the Octane preview, it will continue to work in future versions.
+
+So why is this a preview and not the final Octane release?
+
+While these APIs are stable, not all Octane features are compatible with popular addons yet. We are also still updating documentation, and important tools like the Ember Inspector do not yet work with some Octane features.
+
+In Ember 3.14, Octane will be the recommended way to use Ember and new apps will have Octane's optional features enabled by default. The [guides](https://octane-guides-preview.emberjs.com/release/) and [tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) will show Octane examples, and codemods will be available to help users migrate to Octane.
+
+The Ember tutorial has already been completely rewritten for Octane, and the easiest way to understand the big picture of Octane, especially if you're already an experienced Ember developer, is to [work through the new tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/).
```

Comment:
```suggestion
The Ember tutorial has already been completely rewritten for Octane, and the easiest way to understand the big picture of Octane, especially if you're already an experienced Ember developer, is to [work through the new tutorial](https://emberjs.com/editions/octane).
```

We should not link to the preview URL, since it will be removed when Octane ships.